Workbooks/Windows Virtual Desktop/WVD Insights/WVDWorkbookV1.workbook

```diff
@@ -974,7 +974,7 @@
               "links": [
                 {
```

Comment:
This should be removed.  We do not want to point the user to the AMA config to avoid a user lopp.

Reply:
Removed.